src/steamship/invocable/package_mixin.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABC
+
+from steamship.invocable import Config
+
+
+class PackageMixin(ABC):
+    def instance_init(self, config: Config):
```

Comment:
Oooh yeah `instance_init` is a really nice example of why formalizing mixins is going to be really nice